c2_store/data/media_codecs_intel_c2_video.xml

```diff
@@ -87,7 +87,7 @@ and updated to vendor media codecs.
     <Encoders>
         <MediaCodec name="c2.intel.avc.encoder" type="video/avc">
             <!-- profiles and levels:  ProfileBaseline : Level41 -->
-            <Limit name="size" min="176x144" max="2048x2048" />
+            <Limit name="size" min="176x144" max="8192x8192" />
```

Comment:
The maximum encoding size for AVC is 4k. Please checkout below link for more details.
https://github.com/intel/media-driver/blob/master/docs/media_features.md#supported-encoding-input-format-and-max-resolution

Reply:
thanks, fixed.